gh-109414: Add some basic information about venvs in the introduction. (#109440)
python/cpython: Doc/library/venv.rst

`pip`_ will install Python packages into a virtual environment
without needing to be told to do so explicitly.

A virtual environment is:

* Used to contain a specific Python interpreter and software libraries and
  binaries which are needed to support a project (library or application).

* Contained in a directory, conventionally either named ``venv`` or ``.venv`` in
  the project directory, or under a container directory for lots of virtual
  environments, such as ``~/.virtualenvs``.

> * Not checked into source control systems such as Git, Mercurial etc.

Reviewer: I don't think that you need to give multiple examples:

```suggestion
* Not checked into source control systems such as Git.
```

* Considered as disposable -- it should be simple to delete and recreate it from
  scratch. You don't place any project code in the environment

> * Not considered as movable or copyable -- you just create a new environment in

Reviewer: ```suggestion
* Not considered as movable or copyable -- you just recreate the same environment in
```
